watson-developer-cloud/discovery-components
# fix: issue with document highlighting offset

#### What do these changes do/fix?
This changes fixes the highlighting of the attributes in CIDocument, you can see it's working properly by running storybook


#### How do you test/verify these changes?
 run yarn storybook and navigating to CIDocument -> default. select defined terms under attributes you should see that everything is highlighted correctly.


#### Have you documented your changes (if necessary)?
Yes
#### Are there any breaking changes included in this pull request?
Yes
